contracts/pcv/lido/EthLidoPCVDeposit.sol

```diff
@@ -168,14 +186,9 @@ contract EthLidoPCVDeposit is PCVDeposit {
             _tokenOne == steth ? int128(0) : int128(1),
             _tokenOne == steth ? int128(1) : int128(0),
             amountIn,
-            0 // minimum accepted amount out
```

Comment:
What was the original logic for not making use of Curve's built in slippage protections and supplying `minimumAcceptedAmountOut` here? Why did we choose to enforce it in our contract instead?

Reply:
I used to be paranoid and defensively check everything in the calling smart contract but the curve pool we target is immutable